dtcm/dominion/map.xml

```diff
@@ -26,14 +26,10 @@
     <kit id="spawn-kit">
         <clear/>
         <item slot="0" unbreakable="true" material="stone sword"/>
-        <item slot="1" unbreakable="true" material="bow">
-            <enchantment>infinity</enchantment>
-        </item>
+        <item slot="1" unbreakable="true" material="bow" enchantment="infinity"/>
```

Comment:
Please revert this change

Reply:
A map may modify an item in its spawn kit in many ways including name, enchantments, unbreakability, team colouring, and otherwise. It is easier to see what enchantments are on that item at a glance when they are not inlined.

It also makes configurations like [this](https://github.com/OvercastCommunity/PublicMaps/blob/main/includes/conquest.xml#L21-L28) less confusing.